src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -64,36 +64,18 @@ public static boolean isBamFileName(final String inputFileName) {
     /**
      * Creates a temp directory with the prefix and optional suffix.
      *
-     * @param prefix       Prefix for the directory name.
-     * @param suffix       Optional suffix for the directory name.
-     * @return The created temporary directory.
-     */
-    public static File tempDir(String prefix, String suffix) {
-        return tempDir(prefix, suffix, null);
-    }
-
-    /**
-     * Creates a temp directory with the prefix and optional suffix.
+     * The directory and any contents will be automatically deleted at shutdown.
```

Comment:
Add a note that the method only works if the temp directory is a File

Reply:
done